internal/vds/vds.go

```diff
@@ -109,10 +110,10 @@ func GetAxis(direction string) (int, error) {
 		case "depth":     return AxisDepth,     nil
 		case "time":      return AxisTime,      nil
 		case "sample":    return AxisSample,    nil
-		default:
-			options := "i, j, k, inline, crossline or depth/time/sample"
-			msg := "invalid direction '%s', valid options are: %s"
-			return -1, fmt.Errorf(msg, direction, options)
+	default:
+		options := "i, j, k, inline, crossline or depth/time/sample"
+		msg := "invalid direction '%s', valid options are: %s"
+		return -1, fmt.Errorf(msg, direction, options)
```

Comment:
This indentation is technically correct. But the change is incomplete. The cases should also be un-indented:
```
switch ...
case ...
case ...
default:
```

It's greatly encourage to fix stuff like this when you find it. Even when it's unrelated to the PR. From experience, that is the only way it's ever getting fixed. Nobody is gonna bother to create issues and stand-alone PRs just with minor fixed like this. But fixing them as part of some other work, like you do here is fast and simple. BUT in order to not distract from what's really going on in commit, these types of fixes should be granted there own small commits like: "Fix incorrect indentation in switch-case" or "Fix whitespace errors". That reduces noice in the diffs of the other commits. 

So complete the indentation and extract that (together with the other indentation changes) in a separate commit. And continue fixing stuff like this when you find it 🎉

Reply:
Thanks for letting me. I didn't add this intentionally. It's the IDE who reformatted the file indentation and I mistakenly push that in the PR. It's good that you mentioned it, I will take care of that next time and add additional commits whenever needed. :)

Reply:
Tip: I _always_ use `git add -p`. Then I see exactly what I add to each commit 👍